README.md

```diff
@@ -2,8 +2,8 @@
 
 [![GitHub license](https://img.shields.io/github/license/m-lab/ndt7-client-cc.svg)](https://raw.githubusercontent.com/m-lab/ndt7-client-cc/main/LICENSE) [![Github Releases](https://img.shields.io/github/release/m-lab/ndt7-client-cc.svg)](https://github.com/m-lab/ndt7-client-cc/releases) [![Build Status](https://app.travis-ci.com/m-lab/ndt7-client-cc.svg?branch=main)](https://app.travis-ci.com/m-lab/ndt7-client-cc) [![codecov](https://codecov.io/gh/m-lab/ndt7-client-cc/branch/main/graph/badge.svg)](https://codecov.io/gh/m-lab/ndt7-client-cc)
 
-**Note**: this project is currently unmaintained due to lack of time. Maybe
-one day we'll find a fix for this issue and resume development...
+**Note**: This project is community supported and provided as-is for users and
+client integrators. Contributions are welcome (PRs and issues).
 
 ![tardis](docs/tardis.gif
   "It's not supposed to make that noise. You leave the brakes on.")
```

Comment:
I don't know if you want to keep the GIF or not. My original intention for the GIF, which depicts a time traveling machine, was to say that, had I had time, I had worked happily on the project but I could not.